.github/workflows/semver-auto.yaml

```
@@ -0,0 +1,37 @@
+name: Add PR semver labels on push
+on: [pull_request_target]
+jobs:
+  go-apidiff:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '1'
+
+    - name: Remove the semver label
+      uses: actions-ecosystem/action-remove-labels@v1
+      with:
+        labels: |
+          semver:patch
+          semver:minor
+          semver:major
+        github_token: ${{ github.token }}
+
+    - name: Checking Go API Compatibility
+      uses: joelanford/go-apidiff@main
```

Please pin to the commit sha of the latest release:

```suggestion
      uses: joelanford/go-apidiff@7037613b8a1dfa41270919a854e3703953c36d8f
```

Dependabot will keep it up to date, while letting us review every change.